tests/modules/sitemap/test-sitemap.php

```diff
@@ -39,6 +39,12 @@ public function test_only_pages() {
 		$posts = $this->setup_posts( 2 );
 		$pages = $this->setup_posts( 2, 0, 'page' );
 
+		// create a new page with a delay so that we can test if the sitemap is created in ASCENDING order.
+		// @issue https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/2217
+		sleep( 1 );
```

Comment:
Are you adding sleep because of the time it takes to process the sitemap? If so, would it be safer to make it 3 seconds?

Reply:
i'm adding sleep so that the post is created with a different timestamp than setup_posts which might have the same timestamp down to the second depending on how fast it creates posts.